.github/dependabot.yml

```
@@ -0,0 +1,16 @@
+# Dependabot configuration.
+#
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Suggest aiming for 9am UTC (on Mondays) so the expected time of the PRs is more consistent.

```suggestion
      interval: "weekly"
      time: "09:00"
```

---

I didn't add it on purpose. If everyone sets a time, especially at a whole hour, this causes a peak burden on the GH servers, which may have unintended side-effects. Leaving the time open, you still know that the PRs should have arrived by midnight UTC on Monday, but it allows GH/Dependabot to sort out their queue with more flexibility.
(not that I know how they sort out their queue, but that's not our problem anyway)

Also note that I've used the same config across all the PHPCompatibility repos, so changing it here would make this config inconsistent with the other configs.